### PR TITLE
Bngp 439 upload biodiversity metric file

### PR DESCRIPTION
**Pull request by shaundefra**
Add metric tests  

and add all filetypes regression test

### CODE REVIEW

File: src/features/step_definitions/upload-documents.steps.js [hunk 1]
```diff
@@ -1,52 +1,115 @@
...
+import managementPlanUploadPage from "../page_objects/management_plan/management-plan-upload.page";
+import managementPlanCheckPage from "../page_objects/management_plan/management-plan-check.page";
+import legalAgreementUploadPage from "../page_objects/legal_agreement/legal-agreement-upload.page";
+import legalAgreementCheckPage from "../page_objects/legal_agreement/legal-agreement-check.page";
+import landBoundaryFileUploadPage from "../page_objects/land_boundary/upload-land-boundary.page";
+import landBoundaryFileCheckPage from "../page_objects/land_boundary/check-land-boundary-file.page";
+import metricUploadPage from "../page_objects/metric/metric-upload.page";
+import metricCheckPage from "../page_objects/metric/metric-check.page";
+import { assert } from "console";
+import { get } from "http";
```
Comment (timstone) [inline src/features/step_definitions/upload-documents.steps.js:12]: Are these imports used?

File: src/features/step_definitions/upload-documents.steps.js [hunk 2]
```diff
@@ -1,52 +1,115 @@
...
 
   // open the upload url page
-  browser.url(uploadPage.path);
+  browser.url(UploadPage.path);
+
+  await UploadPage.govFileUpload.setValue(remoteFilePath);
+  await UploadPage.continueButton.click();
+});
+
+Given(/^I should be able to upload all allowed filetypes$/, async (table) => {
```
Comment (timstone) [inline src/features/step_definitions/upload-documents.steps.js:64]: Does this need to be a regex?

File: src/features/upload-documents.feature [hunk 3]
```diff
@@ -82,3 +88,26 @@ Feature: upload documents
...
+            | land-boundary   | pdf      |
+            | land-boundary   | jpg      |
+            | metric          | xlsx     |
+            | metric          | xlsm     |
+
+
+
+
+
+
```
Comment (timstone) [inline src/features/upload-documents.feature:113]: about time you sorted out some linting!

File: src/features/step_definitions/upload-documents.steps.js (hunk 2 above)
Comment (shaundefra) [reply src/features/step_definitions/upload-documents.steps.js:64]: will change it

File: src/features/step_definitions/upload-documents.steps.js (hunk 1 above)
Comment (shaundefra) [reply src/features/step_definitions/upload-documents.steps.js:12]: will remove them

**Pull request merged**